src/Main.java

```diff
@@ -1,5 +1,86 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello, World!");
+        System.out.println("Задача 1");
+        int plannedCapital = 2459000;
+        int monthlyAccumulation = 15000;
+        int totalAccumulation = 0;
+        int months = 0;
+
+        while (totalAccumulation < plannedCapital) {
+            totalAccumulation += monthlyAccumulation;
+            months++;
+            System.out.println("Месяц " + months + ", сумма накоплений равна " + totalAccumulation + " рублей");
+        }
+
+        System.out.println("Задача 2");
+        int start = 1;
+        while (start <= 10) {
+            System.out.print(start + " ");
+            start = start + 1;
+        }
+        System.out.println();
+            for (int start2 = 10; start2 >= 1; start2--) {
```

Comment:
```suggestion
            for (start = 10; start >= 1; start--) {
```
переменные с цифрами на концах - это всегда плохо. Сигнал, что с переменными идёт неправильная работа. Можно просто переиспользовать уже созданную переменную вместо создания новой с непонятным названием